src/lib/ngsiNotify/senderThread.h

```diff
@@ -2,47 +2,63 @@
 #define SENDERTHREAD_H
```

Comment:
Actually, in this particular function **format** was needed for statistics, the call to *statisticsUpdate*